HiP-DataStore.Model/Rest/ReviewArgs.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Rest
+{
+    public class ReviewArgs
+    {
+
+        public string Description { get; set; }
```

Comment:
We should be able to merge ReviewArgs and ReviewUpdateArgs by using default values.